ci/jenkins/test-vmc.sh

```diff
@@ -38,6 +38,13 @@ TEST_FAILURE=false
 CLUSTER_READY=false
 DOCKER_REGISTRY=""
 CONTROL_PLANE_NODE_ROLE="master|control-plane"
+provider="vsphere"
```

Comment:
Now that new parameters and variables are added for CAPA, maybe we can rename this file to from test-vmc.sh to test-clusterAPI.sh